packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -89,10 +90,31 @@ export async function runBrokenLinks(
     await origin_page.waitForSelector(options.wait_for_selector);
   }
 
+  // TODO check origin_link
+
+  if (options.wait_for_selector) {
```

Comment:
This seems to be a repeat of code on line 88?

Reply:
fixed (merge/rebase error)